Source/Harmony/CompFlickable.cs

```diff
@@ -0,0 +1,30 @@
+﻿using HarmonyLib;
+using RimWorld;
+using Verse;
+
+namespace OpenTheWindows
+{
+    [HarmonyPatch(typeof(CompFlickable), nameof(CompFlickable.DoFlick))]
+    public static class CompFlickable_DoFlick
+    {
+        public static void PostFix(CompFlickable __instance)
```

Comment:
This uppercase F caused this patch to silent fail, and this cascaded to every window not being able to open or close! It's been months and people only complained about minor things, even thoug the issue was huge. It was really hard to track down what was going on, so I just HAD to write this! :-P

Reply:
Story of my life, my bd